### PR TITLE
Component (typography)

### PR DESCRIPTION
**Pull request by ricardovera76**
Created Necessary Files for Typography Atom component and functionality

### CODE REVIEW

File: src/components/atoms/Button/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      return `bg-transparent text-black dark:text-white py-2 px-4 border-2 border-black dark:border-white`;
+    },
+    text: function () {
+      return `bg-transparent text-black dark:text-white py-2 px-4 hover:bg-primary hover:text-white`;
+    },
+    custom: function () {
+      return '';
+    },
+  };
+  return (buttonClases[buttonVariant] || buttonClases['normal'])();
```
Comment (jhonderquintero) [inline src/components/atoms/Button/Button.tsx:24]: If you are setting buttonVariant with a defaultValue === 'normal'. Is not necessary the conditional in the return, 'cause always the first sentence in the OR operator will return true.

**Review by jhonderquintero — CHANGES_REQUESTED** [review]
I left some comments, feel free to reply if you have any better idea on what i purpose.

File: src/components/atoms/Button/Button.tsx (hunk 1 above)
Comment (ricardovera76) [reply src/components/atoms/Button/Button.tsx:24]: ok, thanks

File: src/components/atoms/Typography/index.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import { styles } from './styles.style';
+
+const index = ({
+  variant = 'text',
+  isBold = false,
+  isAlt = false,
+  children,
+}: TypographyProps) => {
+  const classes = `${getTypographyClasses(variant, isBold, isAlt)}`;
+  if (variant.indexOf('heading') !== -1) {
```
[the remark this answers is not in the document]
Comment (ricardovera76) [reply src/components/atoms/Typography/index.tsx:12]: to check what size of a heading will be used eg.: heading, heading-lg, heading-sm, each one will represent a h[x] semantic tag

File: src/components/atoms/Typography/styles.style.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
+const textLg = 'font-Nunito text-2xl color-slate-100 dark:text-white';
```
[the remark this answers is not in the document]
Comment (ricardovera76) [reply src/components/atoms/Typography/styles.style.ts:1]: 🤔 ok

**Review by jhonderquintero — APPROVED** [review]

**Pull request merged**